tutorials/applications/QuantumEvolutionKernel_Tutorial.ipynb

```diff
@@ -0,0 +1,1377 @@
+{
```

Comment:
<p>*and choose the best one</p><p> </p><p>(Also, this list renders well on the docs, you should replicate the formatting on the ones that don't)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>